hardware/products/ucm/intro.md

```diff
@@ -28,20 +27,56 @@ The peripherals found on each module may vary.
 | VBAT (Battery Backup for RTC)                         | 1         |
 | JTAG                                                  | 1         |
 
+> ### Peripherals explained
+> #### UART (Universal Asynchronous Receiver/Transmitter) -
+> #####UART is an asynchronous serial communication method to allow the user the ability to configure the speed and format of the data sent. [*Example TinyCLR code*](../../../tinyclr/tutorials/uart.md)
```

Comment:
For the example code, make the header above for the interface a link like: ### [UART](../...../uart.md) (Universal Asynchronous Receive/Transmitter)

Reply:
I think that there should be something to imply the link goes to TinyCLR code or at least some kind of explanation that implies the link will go to TinyCLR code.